main.go

```diff
@@ -47,8 +47,12 @@ var exitFirst = flag.Bool("x", false, "exit on the first failure")
 var errorShown = false
 var usage func(string)
 
-func displayUnknown(name string) {
-	fmt.Printf("%s ?\n", name)
+func displayUnknown(tmpl *template.Template, name string, ref *retrodep.Reference, root string) {
```

Comment:
Confused about what the parameters are here. What's name, and what's root?

I'd expect "topLevelMarker, name string", since that's what's getting passed in -- the optional "*" marker, and the name of the package. BUT: we never showed the topLevelMarker before, so there is no need to do that. Just `"%s ?\n", name` will keep compatibility, as long as name is the name of the package.

Reply:
The `name` should have been `topLevelMarker`. The signature of `displayUnknown()` was reciprocated from `display()` (where the same contents to name are passed), only the `root` was added to make it compatible.

I'll see what can I do, to make it more clear.

Reply:
I've changed the signature to:
```
displayUnknown(tmpl *template.Template, topLevelMarker string, ref *retrodep.Reference, projectRoot string) 
```
See 83baca8 and e4c3042.